sdk/core/Azure.Core/src/Shared/DelayStrategy.cs

```diff
@@ -1,27 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-#nullable enable
-
-using System;
-
-namespace Azure.Core
-{
-    /// <summary>
-    /// Strategy to control delay behavior.
-    /// </summary>
-    internal abstract class DelayStrategy
```

Comment:
You'll have to fixup all these references https://grep.app/search?q=DelayStrategy&filter[repo][0]=Azure/azure-sdk-for-net&filter[lang][0]=XML

This may also affect the generator, but not sure.

Reply:
There will need to be some generator changes for this if we replace the shared strategies with new public ones.

Reply:
Might need to leave these here temporarily until we can make the autorest.csharp changes then can remove in a follow up